src/screens/PinFlow/ConfirmTransactionPassword.tsx

```diff
@@ -134,6 +134,7 @@ const styles = StyleSheet.create({
   inputItemContainer: {
     paddingTop: 20,
     width: '100%',
+    height: 100,
```

Comment:
what does mean `height: 100` without any unit? How does it work?

Reply:
px, a constant value of height to prevent jumping this icon visibility

Reply:
k, thanks 😊